Makefile_rke2

```diff
@@ -49,7 +49,7 @@ attack: copy-scripts create-bad exec
 redpanda: helm 
 	-$(HELM) repo add redpanda https://charts.redpanda.com
```

Comment:
I just tried to deploy the changes on RKE and the repo needs to be renamed to `redpanda-data`.

Reply:
good catch, thanks